server/features/search_events.feature

```diff
@@ -174,10 +174,12 @@ Feature: Event Search
         ]}
         """
         When we get "/events_planning_search?repo=events&only_future=false&language=en"
-        Then we get list with 1 items
+        Then we get list with 3 items
```

Comment:
This change doesn't really test the `language=en` filter now that the system default is applied on creation.
It might be good to change this Event (at the top of this file) and this URL to use a language different to the default (such as `fr-CA`) instead